src/screens/PathsList.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React from 'react';
+import { ScrollView, Text, View } from 'react-native';
```

Comment:
nit: new line between absolute and relative imports

Reply:
I am thinking to use [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import) for doing that, not pretty sure if it will work